tools/swift-format/Sources/Rules/OperatorWhitespace.swift

```diff
@@ -1,15 +1,60 @@
-import Core
-import Foundation
-import SwiftSyntax
-
-/// Exactly one space must appear before and after each binary operator token.
-///
-/// Lint: If an invalid number of spaces appear before or after a binary operator, a lint error is
-///       raised.
-///
-/// Format: All binary operators will have a single space before and after.
-///
-/// - SeeAlso: https://google.github.io/swift#horizontal-whitespace
-public final class OperatorWhitespace: SyntaxFormatRule {
-
-}
+ import Core
+ import Foundation
+ import SwiftSyntax
+ 
+ /// Exactly one space must appear before and after each binary operator token.
+ ///
+ /// Lint: If an invalid number of spaces appear before or after a binary operator, a lint error is
+ ///       raised.
+ ///
+ /// Format: All binary operators will have a single space before and after.
+ ///
+ /// - SeeAlso: https://google.github.io/swift#horizontal-whitespace
+ public final class OperatorWhitespace: SyntaxFormatRule {
+  public override func visit(_ token: TokenSyntax) -> Syntax {
+    guard let nextToken = token.nextToken else { return token }
+    let exceptions =  ["...", "..<", ">.."]
```

Comment:
This might work better as a constant global outside this function. Not sure what the Swift optimizer will do, but `visit(_ token: TokenSyntax)` is a pretty hot function, and this allocation can be avoided each call.